torch/ao/quantization/experimental/APoT_tensor.py

```diff
@@ -2,11 +2,11 @@
 from torch.ao.quantization.experimental.quantizer import APoTQuantizer
 
 # class to store APoT quantized tensor
-class TensorAPoT(torch.Tensor):
+class TensorAPoT():
```

Comment:
there was a comment in the previous PRs suggesting to store the quantized tensor in TensorAPot, but I don't see this being done in this PR?

Reply:
TensorAPoT should store the quantized tensor i.e., the output of your `quantize_apot` method

Reply:
@dzdang yep but `quantize_apot` needs a tensor2quantize and quantizer doesn't store data -- i guess then tensor2quantize should get passed in to my `init` function for apot_tensor?

Reply:
not sure what the issue is? quantize_apot returns the quantized tensor which you can store in apot_tensor

Reply:
should the parameter that `quantize_apot` needs be passed to it by the `init` function in apot_tensor?

Reply:
yes that works or we can do this in a 2 step process:
1) construct apot tensor with quantizer and leave data field uninitialized
2) create a quantize method in apot_tensor that takes in fp tensor that calls quantizer->quantize_apot

Reply:
ok sounds great, will do